src/bypass_url_parser/__init__.py

```diff
@@ -632,7 +636,7 @@ def _save_results(self, url_obj):
             # Json logfile - Starting at SaveLevel.PERTINENT
             if self.save_level >= self.SaveLevel.PERTINENT:
                 json_file = f"{outdir}{Tools.SEPARATOR}{Bypasser.DEFAULT_JSON_FILENAME}"
-                with open(json_file, mode="wt", encoding=self.encoding) as file:
+                with open(json_file, mode="w", encoding=self.encoding) as file:
```

Comment:
We'll have to check that this doesn't introduct breaking changes.. :sweat_smile: 
And test that though installs from pdm, pip, pipx.. X_X

Reply:
The `t` doesn't get in the way but it's not essential here either, it's the default behaviour :

![image](https://github.com/laluka/bypass-url-parser/assets/15103453/236284dc-54f8-4f42-9380-b2fe66bc04ac)

Reply:
Yep, all changes in this file should be safe, they were produced by `ruff`, sorry for the noise 😅 

Only manual changes are in `load_file_into_memory_list`